packages/@vuepress/markdown-loader/index.js

```diff
@@ -42,7 +43,11 @@ module.exports = function (src) {
 
   if (!isProd && !isServer) {
     const inferredTitle = inferTitle(frontmatter.data, frontmatter.content)
-    const headers = extractHeaders(content, ['h2', 'h3'], markdown)
+    let headersToExtract = ['h2', 'h3']
+    if (siteConfig.markdown && siteConfig.markdown.extractHeaders) {
+      headersToExtract = siteConfig.markdown.extractHeaders
+    }
+    const headers = extractHeaders(content, headersToExtract, markdown)
```

Comment:
I remember I have said that `siteConfig` shouldn't be passed to ·`markdown-loader`

Reply:
Should I just change this to pass in `siteConfig.markdown` instead of all of `siteConfig`?